includes/storage/SQLStore/SMW_SQLStore3.php

```diff
@@ -702,7 +702,7 @@ public function clear() {
 	 *
 	 * @param string $connectionTypeId
 	 *
-	 * @return mixed
+	 * @return SMW\MediaWiki\Database
```

Comment:
This uses relative path. The doc will break when the class is put in a NS. So it's better to use the FQN or do an import.